mmcv/image/geometric.py

```diff
@@ -27,7 +27,7 @@ def _scale_size(
     Returns:
         tuple[int]: scaled size.
     """
-    if isinstance(scale, (float, int)):
+    if isinstance(scale, float):
```

Comment:
Hi @MambaWong , `scale` should support accepting `int` type so we only need to update the docstring.

Reply:
I'm confused about the need for `int`, can you give me an example.

Reply:
Hi, this is just for the convenience of the user, for example, if you want to double the size of the image, you can pass in 2, but of course, you can also pass in 2.0.